### DISCUSSION TITLE
How do we get combined coverage reports that include instrumented ui tests?

### DISCUSSION THREAD

**Discussion by neiljaywarner** [insideapp-oss/sonar-flutter#48] [Q&A]
1) how can we get combined coverage reports that include instrumented ui tests?

2) Are widget tests included by default?

**Comment by zippy1978** [comment]
Hi @neiljaywarner,

Have a look at https://github.com/insideapp-oss/sonar-flutter/issues/184

**Comment by neiljaywarner** [comment]
The last comment said they aren't visible?

And if it's in develop when will it come to stable

....sent from my phone
